index.template.html

```diff
@@ -2112,7 +2112,9 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
     instance of <code>SecurityScheme</code>, MUST be serialized as a JSON object.
   </span></p>
 
-  <p><span class="rfc2119-assertion" id="td-security-schemes">
+  <p><span class="at-risk">The following feature is at
+    risk: </span>
+      <span class="rfc2119-assertion" id="td-security-schemes">
```

Comment:
This feature has been a part of TD version 1.0 (see https://www.w3.org/TR/wot-thing-description/#security-serialization-json), can it still be marked as at risk?

(I have to admit that I do not fully understand what this assertion is actually mandating, though. Maybe it could be rephrased if it is worth keeping?)

Reply:
Thanks for the hint. I removed the at-risk label again.